application/minedtx/mined_test.go

```diff
@@ -274,6 +317,135 @@ func TestMined(t *testing.T) {
 	}
 }
 
+func TestMined2(t *testing.T) {
```

Comment:
Can we get a more descriptive name here?

Reply:
I will use this one to overwrite the previous `TestMined` test.